rebound/horizons.py

```diff
@@ -112,7 +112,7 @@ def getParticle(particle=None, m=None, x=None, y=None, z=None, vx=None, vy=None,
     elif idn is not None:
         try:
             p.m = float(re.search(r"BODY%d\_GM .* \( *([\.E\+\-0-9]+ *)\)"%int(idn), HORIZONS_MASS_DATA).group(1))
-            p.m *= KM2AU*KM2AU*KM2AU/(S2YRTWOPI*S2YRTWOPI)
+            p.m /= 6.674e-20 # divide by G (horizons masses give GM)
```

Comment:
It's G in the units used in the table of GM values.  I've made it more transparent by defining a G constant